rfcs/0005-pablo-distribution.md

```diff
@@ -0,0 +1,667 @@
+given staking reward pool,
+
+Pre-defined reward rate (say per second) <img src="0005-pablo-distribution-assets/images/stem-6fb32a8803a6d58cd54908033a2556f9.png" width="23" height="6" alt="stem 6fb32a8803a6d58cd54908033a2556f9" /></span>
+
+Pre-defined reward calculation epoch in seconds <img src="0005-pablo-distribution-assets/images/stem-6184b58307a1dc90934a6a7051a42ceb.png" width="22" height="8" alt="stem 6184b58307a1dc90934a6a7051a42ceb" /></span>
+
+Reward per calculation epoch <img src="0005-pablo-distribution-assets/images/stem-b219ff7e7a0df744c99c2e11229a1ded.png" width="33" height="8" alt="stem b219ff7e7a0df744c99c2e11229a1ded" /></span>
+
+Previous total reward pool before the current epoch <img src="0005-pablo-distribution-assets/images/stem-53fadade13e71b863963af9a23b28b71.png" width="25" height="8" alt="stem 53fadade13e71b863963af9a23b28b71" /></span>
+
+Assuming there is a per epoch calculation which adds to the pool, the
+total reward pool for the current epoc,
+
+<img src="0005-pablo-distribution-assets/images/stem-667bfb2c3da043fcfff3288c44c1cc6e.png" width="103" height="10" alt="stem 667bfb2c3da043fcfff3288c44c1cc6e" /></span>
+
+Reward pool shares for <img src="0005-pablo-distribution-assets/images/stem-55a049b8f161ae7cfeb0197d75aff967.png" width="9" height="6" alt="stem 55a049b8f161ae7cfeb0197d75aff967" /></span>
+stakers,
+
+<img src="0005-pablo-distribution-assets/images/stem-df3438a6dae343911942f03a3f3e1150.png" width="52" height="12" alt="stem df3438a6dae343911942f03a3f3e1150" /></span>
+
+Where <img src="0005-pablo-distribution-assets/images/stem-39e8c7852cdbd74b28d331353778e128.png" width="21" height="9" alt="stem 39e8c7852cdbd74b28d331353778e128" /></span>
+staker share is <img src="0005-pablo-distribution-assets/images/stem-aabe1517ce1102595512b736cbf264bb.png" width="14" height="7" alt="stem aabe1517ce1102595512b736cbf264bb" /></span>
+
+Existing <img src="0005-pablo-distribution-assets/images/stem-39e8c7852cdbd74b28d331353778e128.png" width="21" height="9" alt="stem 39e8c7852cdbd74b28d331353778e128" /></span>
+staker reward,
+
+<img src="0005-pablo-distribution-assets/images/stem-e1359ae7d0fae29ebf9e42efcaa5536e.png" width="111" height="18" alt="stem e1359ae7d0fae29ebf9e42efcaa5536e" /></span>
+
+##### 5.3.1.1. When adding a new staker <span class="image"><img src="0005-pablo-distribution-assets/images/stem-0b46f732c83c0e66067b0e50c2156089.png" width="29" height="8" alt="stem 0b46f732c83c0e66067b0e50c2156089" /></span>, existing stakers(<span class="image"><img src="0005-pablo-distribution-assets/images/stem-55a049b8f161ae7cfeb0197d75aff967.png" width="9" height="6" alt="stem 55a049b8f161ae7cfeb0197d75aff967" /></span>) reward would be,
+
+<img src="0005-pablo-distribution-assets/images/stem-569c4bf984a23f18046277fd561e89a3.png" width="126" height="20" alt="stem 569c4bf984a23f18046277fd561e89a3" /></span>
+
+As this is less than what is expected above, an adjustment " delta P "
```

Comment:
not sure why not to render as $delta_P$?

Reply:
corrected